codes/php/chapter_computational_complexity/space_complexity.php

```diff
@@ -0,0 +1,104 @@
+<?php
+/**
+ * File: time_complexity.php
+ * Created Time: 2024-02-27
+ * Author: skating (121826725@qq.com)
+ */
+
+/* 函数 */
+function func() {
+    // 执行某些操作
+    return 0;
+}
+
+const a = 0;    // 常量
```

Comment:
Could we define it in the function `constants`?

Reply:
In PHP, constants can be defined individually or as attributes of a class, but they cannot be defined in methods.